pkg/agent/endpoints/sds/handler.go

```diff
@@ -54,6 +55,10 @@ type Handler struct {
 	}
 }
 
+func (h *Handler) DeltaSecrets(discovery_v2.SecretDiscoveryService_DeltaSecretsServer) error {
+	return errors.New("not implemented")
```

Comment:
Should this be gRPC error code UNIMPLEMENTED ?

Reply:
Updated